scrapers_next/fl/people.py

```diff
@@ -168,7 +168,7 @@ def process_item(self, item):
         name = item.xpath("./a/div[@class='team-txt']/h5/text()")[0].strip()
 
         # skips empty chairs due to pending regular/special election
-        if "pending, special" or "pending, election" in name.lower():
+        if "pending, special" in name.lower() or "pending, election" in name.lower():
```

Comment:
Better formatting would be `if any(s in name.lower() for s in ["pending, special", "pending, election"])`.